File: lab1.cpp
#include <iostream>
using namespace std;

int main () 
{
	double height, rad; //висота та радіус заданих фігур
	double cylVolume, conVolume; //об'єми циліндра та конуса
	char end;
	
	cout << "Enter height and radius of the objects \nHeight: ";
	cin >> height;
	cout << "Radius: ";
	cin >> rad;
	
	// обчислення об'ємів
	cylVolume = 3.1415*(rad*rad)*height;
	conVolume = cylVolume/3;
	
	printf("Volume of the cylinder is %.3f\n", cylVolume);
	printf("Volume of the cone is %.3f\n", conVolume);
	cin >> end;
}

File: lab1/lab1.py
# введення радіусу та висоти об'єктів

rad = float(input("Enter height and radius of the objects \nRadius: "))
height = float(input("Height: "))

# обчислення об'єму циліндра та конуса

cylVolume = rad*rad*3.1416*height
conVolume = cylVolume/3

print ("Volume of the cylinder is %.3f" % cylVolume)
print ("Volume of the cone is %.3f" % conVolume)
a = input ("")